### PR TITLE
Add parametrized tests

### PR DESCRIPTION
**Pull request by alexander-kulikovskii**
PRs must be submitted under the terms of our Contributor License Agreement [CLA](https://github.com/mapbox/mapbox-maps-android/blob/main/CONTRIBUTING.md#contributor-license-agreement).

## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [x] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [ ] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog></changelog>`.

### User impact (optional)

This PR adding parametrized tests where applicable.

### CODE REVIEW

File: sdk-base/src/test/java/com/mapbox/maps/util/MathUtilsTest.kt [hunk 1]
```diff
@@ -1,42 +1,60 @@
...
+import com.mapbox.maps.util.MathUtils.prepareOptimalBearingPath
+import com.mapbox.maps.util.MathUtils.shortestRotation
 import org.junit.Assert.assertArrayEquals
 import org.junit.Assert.assertEquals
 import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
 
-internal class MathUtilsTest {
+private const val EPS = 0.0001
```
Comment (kiryldz) [inline sdk-base/src/test/java/com/mapbox/maps/util/MathUtilsTest.kt:11]: put in companion object block?

**Review by kiryldz — APPROVED** [review]
LGTM with a small nit.
Congrats with a first PR here @alexander-kulikovskii!

File: sdk-base/src/test/java/com/mapbox/maps/util/MathUtilsTest.kt (hunk 1 above)
Comment (alexander-kulikovskii) [reply sdk-base/src/test/java/com/mapbox/maps/util/MathUtilsTest.kt:11]: no need I guess. I used this constant in two different classes inside one file

File: extension-localization/src/test/java/com/mapbox/maps/extension/localization/SupportedLanguagesTest.kt [hunk 2]
```diff
@@ -18,8 +18,10 @@ private val commonLanguageList = listOf(
   arrayOf(NAME_JA, Locale.JAPANESE),
   arrayOf(NAME_KO, Locale.KOREA),
   arrayOf(NAME_ZH_HANS, Locale.SIMPLIFIED_CHINESE),
-  arrayOf(NAME_ZH_HANS,
```
Comment (axti) [inline extension-localization/src/test/java/com/mapbox/maps/extension/localization/SupportedLanguagesTest.kt]: Should it be amended with previous commit?

**Review by Chaoba — APPROVED** [review]
LGTM

File: extension-localization/src/test/java/com/mapbox/maps/extension/localization/SupportedLanguagesTest.kt (hunk 2 above)
Comment (alexander-kulikovskii) [reply extension-localization/src/test/java/com/mapbox/maps/extension/localization/SupportedLanguagesTest.kt]: Maybe I miss some doc, but why it should be amended with prev commit?

Comment (kiryldz) [reply extension-localization/src/test/java/com/mapbox/maps/extension/localization/SupportedLanguagesTest.kt]: General rule is to have easy-to-read commit history but in general they're all squashed when PR is merged to `main`.

**Review by axti — APPROVED** [review]
Nice job done 👍

**Pull request merged**